MIGRATION.md

```diff
@@ -864,6 +865,43 @@ import type { SvelteRenderer } from '@storybook/svelte';
 // etc.
 ```
 
+#### Renamed `DecoratorFn` to `Decorator`
+
+In 6.x you could import the type `DecoratorFn`:
+
+```ts
+import type { DecoratorFn } from '@storybook/react';
+// etc.
+```
+
+This type is deprecated in 7.0, instead you can use the type `Decorator`:
+
+```ts
+import type { Decorator } from '@storybook/react';
+// etc.
+```
+
+The type `Decorator` accepts a generic parameter `TArgs`. This can be used like this:
+
+```tsx
+import type { Decorator } from '@storybook/react';
+
+const withTheme: Decorator<{ theme: 'light' | 'dark' }> = (Story, { args }) => (
+  <ThemeProvider theme={args.theme}>
+    <Story />
+  </ThemeProvider>
+);
+```
+
+If you want to use `Decorator` in a backwards compatible way to `DecoratorFn`, you can use:
+
+```tsx
+import type { Args, Decorator } from '@storybook/react';
+
+// same Decorator<Args> is the same as the old DecartorFn
```

Comment:
```suggestion
// Decorator<Args> behaves the same as the DecoratorFn<Args>
```

Reply:
Sounds good, it would then become:

```ts
// Decorator<Args> behaves the same as the DecoratorFn
```

As the old DecoratorFn had no generics.